datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -243,6 +268,14 @@ private static void configureDatasetResolvers(final RuntimeWiring.Builder builde
                                     DATASET_TYPE,
                                     (env) -> ((RelatedDataset) env.getSource()).getDataset().getUrn()))
                     )
+            )
+            .type("GenericLineageRelationship", typeWiring -> typeWiring
```

Comment:
If we have this, do we also need the resolver above? (genericDownstreamLineage)

Reply:
I found that I did- I want to investigate this more async but I was not seeing any chart data resolved without this